### PR TITLE
feat: upgrade network agent version

### PR DESCRIPTION
**Pull request by angao**
**What type of PR is this?**
/kind upgrade
/area network

**What this PR does / why we need it**:

**Which issue(s) this PR is related to** *(optional, link to 3rd issue(s))*:

Fixes #

Reference to #

**Special notes for your reviewer**:

/cc @hanxueluo @WangMinjie114 

**Does this PR introduce a user-facing change?**:

```release-note
Upgrade network agent image version
```

### CODE REVIEW

File: inventory/deploy-cluster/group_vars/all/download.yml [hunk 1]
```diff
@@ -27,9 +27,9 @@ pod_infra_version: "3.2"
 nginx_image_tag: 1.19
 dnsautoscaler_version: 1.8.1
 coredns_version: "1.6.7"
-fixedip_ipam_version: "v1.5.0-rc.1"
-bridge_vlan_image_tag: "v1.5.0-alpha.3"
-network_agent_image_tag: "v1.5.0-alpha.3"
+fixedip_ipam_version: "v1.5.0-alpha.2"
```
Comment (hanxueluo) [inline inventory/deploy-cluster/group_vars/all/download.yml]: fixedip_ipam_version ?

Comment (angao) [reply inventory/deploy-cluster/group_vars/all/download.yml]: done, 刚解决冲突整错了

**Pull request merged**